src/server/game/Server/Packets/QueryPackets.cpp

```diff
@@ -106,6 +106,21 @@ void WorldPackets::Query::QueryPlayerName::Read()
         _worldPacket >> Hint.NativeRealmAddress.Value;
 }
 
+ByteBuffer& operator<<(ByteBuffer& data, WorldPackets::Query::PlayerGuidLookupHint const& lookupHint)
```

Comment:
Note to self: Use this in https://github.com/TrinityCore/TrinityCore/blob/6.x/src/server/game/Server/Packets/QueryPackets.cpp#L100